obal/data/roles/nightly/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Build a nightly package"
```

Comment:
Feels like a role might be overkill and this could be done in the nightly playbook.

Reply:
I wasn't entirely sure how it would work with "hosts" and variables.